lib/u3d/unity_runner.rb

```diff
@@ -36,6 +36,7 @@ def run(installation, args, raw_logs: false)
         log_file = installation.default_log_file
       end
 
+      Utils.ensure_dir File.dirname(log_file)
```

Comment:
I believe it is more of a cross platform feature, when Unity has never been run on the system. But I faced it with my shinny new Linux so that's why it is there. Should I create a separated PR?